src/components/Game.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react'
+import Board from './Board'
+
+//component
+const Game: React.FC = () => {
+  const initialBoard = ['', '', '', '', '', '', '', '', '']
+  const [history, setHistory] = useState<string[][]>([initialBoard])
+  const [isXNext, setIsXNext] = useState<boolean>(true)
+  const status = 'Next Player: ' + (isXNext ? 'X' : 'O')
+
+  /**
+   * based on the range value, sets the game to history[n]
+   * @param n
+   */
+  function jumpTo(n: number) {
```

Comment:
```suggestion
  const jumpTo = (historyIndex: number) => {
```

I would never use single character variables, always try to be descriptive and concise.

I'd also just go ahead and use arrow functions everywhere instead of using the function syntax.